contract/src/store.cairo

```diff
@@ -0,0 +1,27 @@
+use dojo::world::WorldStorage;
```

Comment:
This file is a standard practice to encapsulate functions that interact with world storage directly. So I took the liberty to add it.